docs/msbuild/errors/msb3644.md

```diff
@@ -36,8 +36,8 @@ The issue is that the `TargetFrameworkIdentifier` is misspelled. It should be `.
 
 ## Reference assemblies folder
 
-Next, note the framework identifier and version requested in the `TargetFramework`, `TargetFrameworkIdentifier`, and `TargetFrameworkVersion` and look for the reference assembly folder for that version on the local drive.  For example, on Windows, the 32-bit reference assemblies for .NET Core should be under *C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETCore* and 64-bit .NET Framework reference assembly folders for should be under *C:\Program Files\Reference Assemblies\Microsoft\Framework\.NETFramework*. If the folder for the version your project needs is not present or appears to be corrupted, you might need to install or reinstall that version. You can install the required version at [Download .NET](https://dotnet.microsoft.com/download/).
+Next, note the framework identifier and version requested in the `TargetFramework`, `TargetFrameworkIdentifier`, and `TargetFrameworkVersion` and look for the reference assembly folder for that version on the local drive.  For example, on Windows, the 32-bit reference assemblies for .NET Core should be under *C:\Program Files (x86)\Reference Assemblies\Microsoft\Framework\\.NETCore* and 64-bit .NET Framework reference assembly folders for should be under *C:\Program Files\Reference Assemblies\Microsoft\Framework\\.NETFramework*. If the folder for the version your project needs is not present or appears to be corrupted, you might need to install or reinstall that version. You can install the required version at [Download .NET](https://dotnet.microsoft.com/download/).
```

Comment:
Due to escaping, the last backslashes in `\.NETCore` and `\.NETFramework` were missing